doc/python/box-plots.md

```diff
@@ -138,6 +181,45 @@ fig = go.Figure(data=[go.Box(y=[0, 1, 1, 2, 3, 5, 8, 13, 21],
 fig.show()
 ```
 
+### Choosing The Algorithm For Computing Quartiles
+
```

Comment:
Can you link to the explanations given above, using the internal anchor link? (it would probably be "/python/box-plots/#difference-between-quartile-algorithms#)

Reply:
Sure, added a link to the explanations above.